src/Notification/Notification.less

```diff
@@ -15,6 +15,7 @@
 	text-align: center;
 	border-radius: @moon-notification-border-radius;
 	background-color: fade(@moon-neutral-bg-color, @moon-notification-background-opacity);
+	-webkit-filter: blur(0);
```

Comment:
Somewhat concerned about this, especially considering that it appears a blur of 0 still applies blur on Retina macbooks and iOS devices (In some cases?  Seems fixed on Blink.  Safari gives odd results:  http://stackoverflow.com/questions/20377990/why-doesnt-blur0-remove-all-text-blur-in-webkit-chrome-on-retina-screens )